PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -54,7 +54,7 @@ defaultSigma=$(echo "sqrt(200)" | bc -l)
 
 #TSC:should --path or --study-folder be the flag displayed by the usage?
 opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects" "--path"
-opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
+opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject label that matches root dir for a single subject experiment. For longitudinal experiment root to be identified correctly, must use longitudinal_timepoint and longitudinal_template options."
```

Comment:
```suggestion
opts_AddMandatory '--subject' 'Subject' 'subject ID' "subject label that matches root dir for a single time point processing. For longitudinal experiment root to be identified correctly, must use longitudinal_timepoint and longitudinal_template options."
```

Reply:
Also, to avoid confusion, esp. in the context of multiple TP's, this might be a good time to change the various structural pipelines to use `--session` rather than `--subject` as the promoted argument label (with `--subject` still supported/recognized through the newOpts functionality).

Reply:
I agree using --session (session-dir?) instead of --Subject would help, if we don't want to dwell on legacy meaning and avoid more confusion with longitudinal mode.

Reply:
Here is an example of a compatibility flag - any extra flags on the very end (note, for optional parameters, you must specify a default value, even if it is the empty string, in order to add compatibility flags after it) will work, but won't be shown in the help info:

https://github.com/Washington-University/HCPpipelines/blob/9cfa37a378dff26fd5aa4fd2aad24d0136830f0f/PostFreeSurfer/PostFreeSurferPipeline.sh#L56

Reply:
This required refactoring for most structural scripts, including those called by PostFreeSurfer. 
I did not refactor those not under PostFreesurfer/scripts, such as MSMSulc.sh or MyelinMap_BC.sh to avoid unknown compatibility issues. I also did not refactor _1res scripts.